### PR TITLE
Add ItemDates into DateOfCreation for photographs

### PR DESCRIPTION
**Pull request by kehh**
- Update source data query to export new column
- Update import mapping spreadsheet with new mapping added
- Update sample source and destination data

### CODE REVIEW

File: importScripts/sampleRecords/photographs.browse.bundles.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         "ca_objects.SpecificLocation.BoxType": {},
         "ca_objects.SpecificLocation.Copied": {
             "convertCodesToDisplayText" : true
-        }
+        },
+        "ca_objects.DateOfCreation": {}
```
Comment (kehh) [inline importScripts/sampleRecords/photographs.browse.bundles.json:10]: This file gets passed to the CA webservice asking for these specific fields. This is how the `photographs.destination.json` is created.

File: importScripts/sampleRecords/photographs.destination.json [hunk 2]
```diff
@@ -26,7 +26,8 @@
       "display_label":"COLONIAL HOSPITAL, ST MARY'S CATHEDRAL &amp; R.C. BISHOPS HOUSE",
       "ca_objects.StoredLocation":"Photographs Room",
       "ca_objects.SpecificLocation.BoxType":"L1, 1999.1006",
-      "ca_objects.SpecificLocation.Copied":"No"
+      "ca_objects.SpecificLocation.Copied":"No",
+      "ca_objects.DateOfCreation":"1863"
```
Comment (kehh) [inline importScripts/sampleRecords/photographs.destination.json:30]: Existing record that now has a date.

File: importScripts/sampleRecords/photographs.destination.json [hunk 3]
```diff
@@ -36,6 +37,48 @@
       "ca_objects.StoredLocation":"Photographs Room",
       "ca_objects.SpecificLocation.BoxType":"L1, 1999.1008",
       "ca_objects.SpecificLocation.Copied":"Yes"
+    },
```
Comment (kehh) [inline importScripts/sampleRecords/photographs.destination.json:40]: New sample records with dates.

**Comment by mostro20** [comment]
Compared the source and destination json files and all expected transformations occurred in the analysed data. All clear, and makes sense.

**Comment by kehh** [comment]
Merged as @mostro20 was happy with this.

**Pull request merged**